flink-clients/src/test/java/org/apache/flink/client/cli/CliFrontendStopTest.java

```diff
@@ -98,24 +103,26 @@ public void testUnknownJobId() throws Exception {
 		JobID jid = new JobID();
 
 		String[] parameters = { jid.toString() };
-		final ClusterClient clusterClient = createClusterClient(true);
+		String expectedMessage = "Test exception";
+		FlinkException testException = new FlinkException(expectedMessage);
+		final ClusterClient<String> clusterClient = createClusterClient(testException);
 		MockedCliFrontend testFrontend = new MockedCliFrontend(clusterClient);
 
 		try {
 			testFrontend.stop(parameters);
 			fail("Should have failed.");
-		} catch (IllegalArgumentException ignored) {
-			// expected
+		} catch (FlinkException e) {
+			assertTrue(ExceptionUtils.findThrowableWithMessage(e, expectedMessage).isPresent());
 		}
 
 		Mockito.verify(clusterClient, times(1)).stop(any(JobID.class));
```

Comment:
I think I have written it on a previous PR: `verify` shouldn't be needed because you already assert if the exception was thrown.